optimism/src/mips/interpreter.rs

```diff
@@ -34,9 +38,9 @@ pub enum RTypeInstruction {
     SyscallWriteOther,            // syscall (Write ?)
     SyscallFcntl,                 // syscall (Fcntl)
     SyscallOther,                 // syscall (Brk, Clone, ?)
-    MoveZero,                     // movz
-    MoveNonZero,                  // movn
-    Sync,                         // sync
+    MoveZero,                     // movz - FIXME: pseudo?
```

Comment:
`movz` is documented as "MIPS32 removed in  Release 6"

Reply:
Added as a comment in the code.